src/App.tsx

```diff
@@ -1,19 +1,55 @@
 import React from 'react';
-import './App.css';
+import './App.scss';
+import { v4 } from 'uuid';
 
 import users from './api/users';
+import todos from './api/todos';
+import { TodoList } from './TodoList';
+import { TodoForm } from './TodoForm';
 
-const App: React.FC = () => {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
-  );
+const todosWithUsers = todos.map(todo => (
+  {
+    ...todo,
+    id: v4(),
+    user: users.find(user => (user.id === todo.userId)) || null,
+  }
+));
+
+type State = {
+  todoListWithUsers: Todo[];
 };
 
+class App extends React.Component<{}, State> {
+  state = {
+    todoListWithUsers: todosWithUsers,
+  };
+
+  selectList = users.map(user => user.name);
```

Comment:
store it in the state. Or don't do it at all. You do 2 map of an array (first here, second at TodoForm). You can map it only on TodoForm and use user.name